src/main/java/oshi/hardware/platform/mac/MacNetworks.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2016 com.github.dblock.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ * enrico[dot]bianchi[at]gmail[dot]com
+ *    com.github.dblock - initial API and implementation and/or initial documentation
+ */
+package oshi.hardware.platform.mac;
+
+import java.net.NetworkInterface;
+import java.net.SocketException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Enumeration;
+import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import oshi.hardware.common.AbstractNetworks;
+import oshi.hardware.stores.HWNetworkStore;
+
+/**
+ *
+ * @author enrico[dot]bianchi[at]gmail[dot]com
+ */
+public class MacNetworks extends AbstractNetworks {
+
+    private static final Logger LOG = LoggerFactory.getLogger(MacNetworks.class);
+
+    private void setNetworkStats(HWNetworkStore netstore) {
+        // TODO: set network stats on Mac
```

Comment:
So, on Mac, the code to get the stats returns an array of structures with the data, so it's inefficient to read the entire array every time just to match the particular interface name.  To fit that code into this structure we'd probably read it once and cache the data in a map with the netstore.getName() as the key.